tests/unit/modules/test_global_catalog.py

```diff
@@ -102,6 +103,7 @@ def test_create_an_intial_global_catalog():
                 stdout=IDCAMS_create_stdout(NAME),
                 stderr="",
             ),
+            [],
```

Comment:
Think you're gonna have to walk me through why we have so many empty arrays in the test executions